gradio/demo.py

```diff
@@ -0,0 +1,274 @@
+import collections
+import os
+import tempfile
+
+from matplotlib import gridspec
+from matplotlib import pyplot as plt
+import numpy as np
+from PIL import Image
+import urllib
+
+import tensorflow as tf
+import gradio as gr
+
+from subprocess import call
+import sys
+
+import requests
+
+url1 = 'https://cdn.pixabay.com/photo/2021/01/28/18/21/beach-5958718_1280.jpg'
+r = requests.get(url1, allow_redirects=True)
+open("town.jpg", 'wb').write(r.content)
```

Comment:
Our models are trained on cityscapes, it might not generalize well on other images tho. Could we use similar images?

Reply:
sure ill change this to something similar

Reply:
changed to similar images

Reply:
For those demo images, is there any licence issue?
Are they free to use?